app/src/test/java/org/simple/clinic/drugs/selectionv2/entry/CustomPrescriptionEntryControllerv2Test.kt

```diff
@@ -93,4 +96,59 @@ class CustomPrescriptionEntryControllerv2Test {
 
     verify(sheet).moveDrugDosageCursorToBeginning()
   }
+
+  @Test
+  @Parameters(method = "params for showing title")
+  fun `when sheet is created then correct title should be populated`(openAs: OpenAs, showNewEntryTitle: Boolean) {
+    whenever(prescriptionRepository.prescription(any())).thenReturn(Observable.never())
+
+    uiEvents.onNext(CustomPrescriptionSheetCreated(openAs))
+
+    if (showNewEntryTitle) {
+      verify(sheet).showEnterNewPrescriptionTitle()
+    } else {
+      verify(sheet).showEditPrescriptionTitle()
+    }
+  }
+
+  @Suppress("Unused")
+  private fun `params for showing title`(): List<List<Any>> {
+    return listOf(
+        listOf(OpenAs.New(patientUuid), true),
+        listOf(OpenAs.Update(prescriptionUuid), false)
+    )
+  }
+
+  @Parameters(method = "params for showing remove button")
+  @Test
```

Comment:
nitpick: can you move `@Parameters()` annotation below `@Test` like the other functions?